programs/scholarship.md

```diff
@@ -45,21 +44,14 @@ The scholarship committee will review each application holistically, taking into
 ## Applicant Eligibility
 Applicants must be current or past PiE participants in their last year of high school, and must intend to pursue post-secondary education (four-year, technical, vocational, etc.).
 
-## Deliverables
-1. **Scholarship application**
-    * Please type your responses to the Application Questions and send them in DOC, DOCX, or PDF format.
-    * Submit this form and your responses to <scholarship@pioneers.berkeley.edu> as attachments to an email with the word “Application” in the subject line.
-2. **Unofficial transcript (1 copy)**
-    * Please request an unofficial transcript from your school counselor.
-    * Electronic copies can be submitted to <scholarship@pioneers.berkeley.edu> as an attachment to an email with the word “Transcript” in the subject line.
-    * Paper transcripts can be turned in during worksessions to the worksession manager, who will deposit it in the scholarship drop box.
-3. **Recommendation form (2 pages) - Optional but recommended**
-    * Please request a letter from a teacher, counselor, PiE mentor, or PiE staff member who has worked with you in the past. We recommend that you approach your advocate as soon as possible to ensure that we receive the best recommendation on your behalf.
-    * Your advocate can submit the letter to <scholarship@pioneers.berkeley.edu> as an attachment to an email with the word “Recommendation” in the subject line.
+## Apply!
+Paper applications are provided for your convenience. Please use the online web application to submit your scholarship application.
+
+* [2016 Scholarship Application]({{ '/assets/scholarship/2016_pie_scholarship_application_form.pdf' | prepend: site.baseurl }})
+* [2016 Recommendation Form]({{ '/assets/scholarship/2016_pie_scholarship_recommendation_form.pdf' | prepend: site.baseurl }})
 
 <center>
-<a href="{{ '/assets/scholarship/2015_pie_scholarship_application_form.pdf' | prepend: site.baseurl }}" class="btn btn-primary">Download Application (.pdf)</a>
-<a href="{{ '/assets/scholarship/2015_pie_scholarship_recommendation_form.pdf' | prepend: site.baseurl }}" class="btn btn-primary">Download Recommendation Form (.pdf)</a>
+<a href="http://django.matthewchang.com/" class="btn btn-primary">Go to the web application! (Sign in with Google)</a>
```

Comment:
Never link to dev instances. A key thing to remember is that once a link is on the web, it is there forever. Also, the security of my dev instance is questionable because of the debug modes. Please remove